examples/medplum-hello-world/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "medplum-hello-world",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "@emotion/react": "11.10.8",
+    "@mantine/core": "6.0.9",
+    "@mantine/hooks": "6.0.9",
+    "@mantine/notifications": "6.0.9",
+    "@medplum/core": "2.0.17",
+    "@medplum/fhirtypes": "2.0.17",
+    "@medplum/react": "2.0.17",
+    "@tabler/icons-react": "2.17.0",
+    "@types/node": "^18.16.2",
```

Comment:
For `dependencies` and `devDependencies`, we use exact version numbers, so we can remove the "^" here.